spec/controllers/cases_controller_spec.rb

```diff
@@ -446,6 +443,26 @@ def sar_closure_params(sar)
       end
     end
 
+    context 'without stubs' do
+      let!(:in_time_case) { create(:accepted_case) }
+      let!(:late_case) { Timecop.freeze(50.days.ago) { create(:accepted_case) } }
+      let!(:closed_case) { create(:closed_case) }
+
+      before do
+        sign_in responder
+      end
+
+      it 'gets my in time cases' do
+        get :my_open_cases, params: { tab: 'in_time' }
```

Comment:
do we have a similar test for incoming cases? (I'm assuming that's what page was reported in the ticket)